runtime/lua/vim/lsp/util.lua

```diff
@@ -1647,7 +1647,8 @@ function M._make_floating_popup_size(contents, opts)
     width = 0
     for i, line in ipairs(contents) do
       -- TODO(ashkan) use nvim_strdisplaywidth if/when that is introduced.
-      line_widths[i] = vim.fn.strdisplaywidth(line)
+      local ok, line_width = pcall(vim.fn.strdisplaywidth, line)
```

Comment:
BRB; The title part has a smarter way of handling this so I'm gonna try to do whatever that's doing instead of this.